modules/ROOT/pages/con_pipeline-pull-request-resources.adoc

```diff
@@ -0,0 +1,125 @@
+[]
+= Pull Request Resource
+
+Pull Request resource represents a pull request event from a source control
+system.
+
+Adding the Pull Request resource as an input to a Task will populate the
+workspace with a set of files containing generic pull request related metadata
+such as base/head commit, comments, and labels.
+
+The payloads will also contain
+links to raw service-specific payloads where appropriate.
+
+Adding the Pull Request resource as an output of a Task will update the source
+control system with any changes made to the pull request resource during the
+pipeline.
+
```

Comment:
@joaedwar: Please try to use the present tense and active voice in the text above.